testnets/shentutestnet/assetlist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "shentutestnet",
+  "assets": [
+    {
+      "description": "The native token of Shentu",
+      "denom_units": [
+        {
+          "denom": "uctk",
+          "exponent": 0
+        },
+        {
+          "denom": "ctk",
+          "exponent": 6
+        }
+      ],
+      "base": "uctk",
+      "name": "Shentu",
+      "display": "ctk",
+      "symbol": "CTK",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png",
```

Comment:
Please use image_sync, pointing back to mainnet ctk. Something like:
```
"image_sync": {
  "chain_name": "shentu",
  "base_denom": "uctk"
},
```

Reply:
added the image_sync field